modules/odo-core-concepts.adoc

```diff
@@ -10,35 +10,20 @@
 `odo` abstracts Kubernetes concepts into terminology that is familiar to developers:
 
 Application::
-A typical application, developed with a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native approach], that is used to perform a particular task.
+A typical application, developed using a link:https://www.redhat.com/en/topics/cloud-native-apps[cloud-native approach], that performs a particular task.
 +
 Examples of _applications_ include online video streaming, online shopping, and hotel reservation systems.
 
 Component::
-A set of Kubernetes resources that can run and be deployed separately. A cloud-native application is a collection of small, independent, loosely coupled _components_. 
+A set of Kubernetes resources that can run and be deployed separately. A cloud-native application is a collection of small, independent, loosely coupled _components_. `odo` uses Devfiles for the definition of _components_.
 +
 Examples of _components_ include an API back-end, a web interface, and a payment back-end.
 
 Project::
-A single unit containing your source code, tests, and libraries. 
+A _project_ helps achieve multi-tenancy, allowing different teams to deploy multiple applications on the cluster. In Kubernetes, a _project_ is represented by a namespace.
```

Comment:
```suggestion
A single collection of code, tests, and libraries that helps achieve multi-tenancy so that different teams can deploy multiple applications on a cluster. In Kubernetes, a _project_ is represented by a namespace.
```
Currently, the first sentence is structured as a complete sentence and includes the term being defined, unlike the other definitions in the list. My suggestion is one possible way to make this sentence parallel in structure with the other definitions and to avoid using `allowing`, which we can't use, per ISG guidelines.

Reply:
I think projects have changed from 2.5, so they are no longer that list of items, but now just match a `Project` in OPenShift and a `Namespace` in Kubernetes, so I have put:

A _project_ helps achieve multi-tenancy, so that different teams can deploy multiple applications on the cluster. In Kubernetes, a _project_ is represented by a `Namespace` resource, while in OpenShift it is a `Project` resource.